tests/forms_tests/tests/test_forms.py

```diff
@@ -2728,7 +2728,8 @@ class Person(Form):
 
     def test_label_has_required_css_class(self):
         """
-        #17922 - required_css_class is added to the label_tag() of required fields.
+        #17922 - required_css_class is added to label_tag() and legend_tag() of
+        required fields.
```

Comment:
We can chop _"#17922"_ when we're adjusting this comment:
```suggestion
        required_css_class is added to label_tag() and legend_tag() of required
        fields.
```